Pipfile

```diff
@@ -5,6 +5,8 @@ verify_ssl = true
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
+pytest-django = "*"
```

Comment:
Why do we need the pytest package?
Don't we use just the pytest-django package?

Reply:
As I understand, PyTest is the framework and pytest-django is a plugin.

Reply:
> No need of pytest

What do you mean?

Reply:
This is enough 
[dev-packages]
flake8 = "*"
pytest-django = "*"

Reply:
The plugin requires pytest indeed, but I find this implicit dependency pretty ugly.
I prefer the explicit requirement in this case, but feel free to choose the other.

Reply:
Did you try it? i tried and there is no  need, I choose not to die on that hill :)